packages/dcos-integration-test/extra/test_metrics.py

```diff
@@ -1,4 +1,5 @@
 import pytest
+import requests
 import retrying
 
 __maintainer__ = 'mnaboka'
```

Comment:
let's update this while we are here. @mnaboka does not maintain this anymore I think :-).

Reply:
@philipnrmn you okay taking this place?

Reply:
sure